lib/src/main/java/space/taran/arklib/index/RustResourcesIndex.kt

```diff
@@ -0,0 +1,73 @@
+package space.taran.arklib.index
+
+import java.nio.file.Path
+import java.nio.file.attribute.FileTime
+typealias ResourceId = Long
```

Comment:
We need to use the same `ResourceId` as in `arklib`, i.e. CRC-32 hash and also filesize.